pruning/architecture/pruning_loop.py

```diff
@@ -6,18 +6,23 @@
 from architecture.construct_dataset import get_dataloaders
 from architecture.construct_model import construct_model, register_models
 from architecture.construct_optimizer import construct_optimizer
+from architecture.pruning_methods.iterators import get_iterator
 import architecture.utility as utility
 from config.main_config import MainConfig
+import numpy as np
 import pandas as pd
 import torch
 from torch import nn
 import torch.nn.utils.prune as prune
 from wandb.sdk.wandb_run import Run
 
 logger = logging.getLogger(__name__)
+PRUNING_CLASSES = (nn.Linear, nn.Conv2d, nn.BatchNorm2d)
```

Comment:
We need to think about a way to add this to Hydra config somehow.

Reply:
I added it to TODO comment, but by moving it as const at the top of the file, it will be clearer which layers it prunes.